pilot/pkg/status/report.go

```diff
@@ -69,7 +69,7 @@ func (r Resource) String() string {
 
 func (r *Resource) ToModelKey() string {
 	// we have a resource here, but model keys use kind.  Use the schema to find the correct kind.
-	found, _ := crd.SupportedSchemas.FindByPlural(r.Resource)
+	found, _ := collections.All.FindByPlural(r.Resource)
```

Comment:
IIUC the plural resource name is not unique across all schemas. For example, `istio/networking/v1alpha3/gateways`, `k8s/networking.istio.io/v1alpha3/gateways`, `k8s/service_apis/v1alpha1/gateways`. It also doesn't seem to be unique within SupportedSchemas either. I'm might be missing something, but wouldn't this lead to inconsistent / correct behavior?

Reply:
I agree. Its small enough fix that I just fixed it in this PR by passing in the full GVR which should be unique